lib/codeowners/cli/warner.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Codeowners
+  module Cli
+    # checks and prints deprecation warnings
+    module Warner
+      class << self
+        def check_warnings
+          check_github_env
+        end
+
+        def warn(msg)
+          puts "[WARNING] #{msg}"
+        end
+
+        protected
+
+        def check_github_env
+          return if ENV['GITHUB_ORGANIZATION'].nil? || ENV['GITHUB_ORGANIZATION'].empty?
+
+          warn 'Usage of GITHUB_ORGANIZATION ENV variable has been deprecated.'\
+            'Use `codeowners-checker config organization <@organization>` instead.'
```

Comment:
What about we bind the suggestion to just make it copy and paste ;)

```suggestion
            'Run `codeowners-checker config organization #{organization}` instead.'
```